feature/client/src/main/java/com/mifos/feature/client/clientDetails/presentation/ClientDetailsScreen.kt

```diff
@@ -0,0 +1,978 @@
+    notes: (Int) -> Unit,
+    pinpointLocation: (Int) -> Unit,
+    survey: (Int) -> Unit,
+    uploadSignature: (Int) -> Unit,
+    loanAccountSelected: (Int) -> Unit,
+    savingsAccountSelected: (Int, DepositType) -> Unit,
+    activateClient: (Int) -> Unit
+) {
+
+    val context = LocalContext.current
+    val scope = rememberCoroutineScope()
+    val state = clientDetailsViewModel.clientDetailsUiState.collectAsStateWithLifecycle().value
+    val client = clientDetailsViewModel.client.collectAsStateWithLifecycle().value
+    val loanAccounts = clientDetailsViewModel.loanAccount.collectAsStateWithLifecycle().value
+    val savingsAccounts = clientDetailsViewModel.savingsAccounts.collectAsStateWithLifecycle().value
+    val showLoading = clientDetailsViewModel.showLoading.collectAsStateWithLifecycle().value
+
+
+    var clientNotFoundError by rememberSaveable { mutableStateOf(false) }
+    var showMenu by remember { mutableStateOf(false) }
+    var showSelectImageDialog by remember { mutableStateOf(false) }
+    var imageUri by remember { mutableStateOf<Uri?>(null) }
+    val snackbarHostState = remember { SnackbarHostState() }
+
+
+    val file = context.createImageFile()
+    val cameraImageUri = FileProvider.getUriForFile(
+        Objects.requireNonNull(context),
+        "com.mifos.mifosxdroid" + ".provider",
+        file
+    )
+    val permissionState = rememberMultiplePermissionsState(
+        permissions = listOf(
+            Manifest.permission.CAMERA
+        )
+    )
+
+    val galleryLauncher = rememberLauncherForActivityResult(
+        contract = ActivityResultContracts.GetContent(),
+        onResult = { uri ->
+            uri?.let {
+                imageUri = it
+                val bitmap = context.contentResolver.openInputStream(uri).use { stream ->
+                    BitmapFactory.decodeStream(stream).asImageBitmap().asAndroidBitmap()
+                }
+                showSelectImageDialog = false
+                clientDetailsViewModel.saveClientImage(clientId, bitmap)
+            }
+        }
+    )
+
+
+    val cameraLauncher =
+        rememberLauncherForActivityResult(contract = ActivityResultContracts.TakePicture()) { status ->
+            if (status) {
+                imageUri = cameraImageUri
+                val bitmap = context.contentResolver.openInputStream(cameraImageUri).use { stream ->
+                    BitmapFactory.decodeStream(stream).asImageBitmap().asAndroidBitmap()
+                }
+                showSelectImageDialog = false
+                clientDetailsViewModel.saveClientImage(clientId, bitmap)
+            }
+        }
+
+    LaunchedEffect(key1 = true) {
+        clientDetailsViewModel.loadClientDetailsAndClientAccounts(clientId)
+    }
+
+    when (state) {
+
+        is ClientDetailsUiState.ShowClientImageDeletedSuccessfully -> {
+            LaunchedEffect(key1 = true) {
+                snackbarHostState.showSnackbar(message = "Client Image Deleted Successfully")
```

Comment:
move to strings.xml, no hard-coded string

Reply:
I have replaced it with stringresource